README.md

```diff
@@ -9,7 +9,7 @@
 
 The Charmed Kafka Operator delivers automated operations management from day 0 to day 2 on the [Apache Kafka](https://kafka.apache.org) event streaming platform. It is an open source, end-to-end, production ready data platform on top of cloud native technologies. 
 
-This operator can be found on [Charmhub](https://charmhub.io/kafka) and it comes with production-ready features such as:
+The Kafka Operator can be found on [Charmhub](https://charmhub.io/kafka) and it comes with production-ready features such as:
```

Comment:
question: For consistency, don't we need to refer to this charm as "Charmed Kafka" everywhere?

Reply:
uhm, I suppose it is similar. Actually in line 17 below, we say "Kafka Operator". I honestly feel that `Charm Kafka Opeartor on Charmhub`, is a bit too much charming :D